index.js

```diff
@@ -1,7 +1,16 @@
 'use strict';
+
+function flatten(arr) {
+	return arr.reduce((previous, current) => {
+		return previous.concat(Array.isArray(current) ? flatten(current) : current);
+	}, []);
+}
+
 module.exports = function (arr) {
-	var rest = [].concat.apply([], [].slice.call(arguments, 1));
```

Comment:
It's because of the `esnext` flag. We might just disable the rule for that line, @sindresorhus wdyt?